widget/playground/src/containers/configContainer/ConfigContainer.tsx

```diff
@@ -67,29 +68,37 @@ export function ConfigContainer(props: PropsWithChildren) {
         <Main>
           <Header />
           <Content>
-            {showBoundaryGuide && (
-              <BoundarySize side="right">
-                <WidthIcon size={16} color="gray" />
-                <Divider size={4} direction="horizontal" />
-                <Typography variant="label" size="medium" color="neutral600">
-                  Max Width: 390 px
-                </Typography>
-              </BoundarySize>
-            )}
-            <BoundaryGuide
-              style={{ borderWidth: showBoundaryGuide ? '1px' : 0 }}>
-              {props.children}
-            </BoundaryGuide>
+            <div>
```

Comment:
Why this `div` needed?

Reply:
To address the issue depicted in the image, a container needs to be created to properly align the texts related to the boundary guide. The container should have a width that fits the dimensions of the boundary guide to ensure correct placement.

<img width="732" alt="image" src="https://github.com/rango-exchange/rango-client/assets/154742563/a23845a0-30e3-418d-be40-212892a829e8">

Reply:
It's kinda weird, but if it's need, please put a comment on top of it and explain the reason because it can be removed in future easily since removing it will cause problem on a specific situation.

Reply:
Done